cookiecutter.json

```diff
@@ -8,5 +8,7 @@
     "mintable": "y",
     "max_supply": "y",
     "max_supply_amount": 10000,
-    "permitable": "y"
+    "permitable": "y",
+    "royalties": "y",
+    "royalty_percentage": 10
```

Comment:
If it's a percentage, probably better to do something like `salePrice * {{ percentage }} / 100` down below in the code

Reply:
Can also use Decimals e.g.

```vyper
convert(convert(salePrice, decimal) * PERCENTAGE, uint256)
```